addon/components/bootstrap-inputs/-search.js

```diff
@@ -0,0 +1,78 @@
+import Ember from 'ember';
+import layout from '../../templates/components/bootstrap-inputs/-search';
+import { PropTypes } from 'ember-prop-types';
+import { BuilderForPropTypes, BuilderForPropDefaults } from 'ember-bootstrap-controls/utils/prop-definition-tools';
+import autoCompleteTypes from '../../utils/auto-complete-types';
+
+export const propDefinitions = {
+  autocomplete: {
+    default: 'on',
+    description: 'Indicates whether the value can be automatically completed by the browser.',
+    type: PropTypes.oneOf(autoCompleteTypes),
+  },
+  disabled: {
+    description: 'Indicates whether the control is disabled',
+    type: PropTypes.bool,
+  },
+  errors: {
+    description: 'An array of EmberData errors to display.',
+    type: PropTypes.arrayOf(PropTypes.string),
+  },
+  help: {
+    description: 'Additonal text to provide additional context to the user that is displayed below the input.',
+    type: PropTypes.string,
+  },
+  label: {
```

Comment:
Even though there is a default, all labels should be required for accessibility.

Reply:
It would just be the type of `PropTypes.string.isRequired` to make this required.

Reply:
addressed